packages/radix-nuxt/src/index.ts

```diff
@@ -3,18 +3,25 @@ import { addComponent, defineNuxtModule } from '@nuxt/kit'
 import type { } from '@nuxt/schema' // workaround for TS bug with "phantom" deps
 import * as components from 'radix-vue'
 
-export default defineNuxtModule({
+export interface ModuleOptions {
+  prefix: string

```

Comment:
Let's make this prefix optional. So when user wanna have empty options, it wont throw ts error 😁

Reply:
All module options are optional in Nuxt by default (`Partial<ModuleOptions>`) 😁